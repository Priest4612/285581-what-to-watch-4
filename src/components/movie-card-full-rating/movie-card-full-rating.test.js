import React from 'react';
import renderer from 'react-test-renderer';

import filmsMock from '../../mocks/test-films-mock.json';

import MovieCardFullRating from './movie-card-full-rating.jsx';


it(`Render MovieCardFullRating`, () => {
  const {rating, scoresCount} = filmsMock[2];
  const tree = renderer
    .create(<MovieCardFullRating
      rating={rating}
      scoresCount={scoresCount}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
