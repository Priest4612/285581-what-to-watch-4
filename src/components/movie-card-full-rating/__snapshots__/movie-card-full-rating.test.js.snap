// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render MovieCardFullRating 1`] = `
<div
  className="movie-rating"
>
  <div
    className="movie-rating__score"
  >
    0.1
  </div>
  <p
    className="movie-rating__meta"
  >
    <span
      className="movie-rating__level"
    >
      Bad
    </span>
    <span
      className="movie-rating__count"
    >
      823 ratings
    </span>
  </p>
</div>
`;
